### DISCUSSION TITLE
Weird scrolling behavior when there's a video in the PR

### DISCUSSION THREAD

**Discussion by DimitarNestorov** [community/community#38683] [Mobile]
### Select Topic Area

Bug

### Body

I need to scroll really fast to get to the top (or tap on the status bar). Once I play the video it's all good. Latest version of GitHub for iOS.

https://user-images.githubusercontent.com/8790386/200882306-96dac501-edee-4c7f-894c-d30930146afd.mov

**Comment by ossanna16** [comment]
Hi there @dimitarnestorov and welcome to our community! Thank you for reporting this 🙂
